### PR TITLE
Home Screen Completed

### PR DESCRIPTION
**Pull request by Jay-cey**
kyiv-omega.vercel.app

### CODE REVIEW

File: app/layout.js
Comment (TressOhi) [inline app/layout.js:1]: You can make the header/nav and the footer components on their own

**Comment by TressOhi** [comment]
Kindly add link to the successful vercel deployment so I can review.

**Comment by Jay-cey** [comment]
> Kindly add link to the successful vercel deployment so I can review.

kyiv-omega.vercel.app

File: app/layout.js
Comment (Jay-cey) [reply app/layout.js:1]: Fixed 💯

Comment (TressOhi) [reply app/layout.js:1]: Awesome! Will get back to you soon. Keep up the good work! 🚀

Comment (TressOhi) [reply app/layout.js:1]: Let's schedule a meet for Saturday.

**Pull request merged**